tests/e2e-leg-6/revivedb-1-node-restore/vdb-to-revive-id/base/setup-vdb.yaml

```diff
@@ -0,0 +1,40 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1beta1
+kind: VerticaDB
+metadata:
+  name: v-revive-1-node-id
+spec:
+  image: kustomize-vertica-image
+  superuserPasswordSecret: su-passwd
+  ignoreClusterLease: true
```

Comment:
When you convert this to a V1 API. You need to set the `vertica.com/include-cluster-lease` annotation as this parm was removed.

Reply:
will do, if you mean `ignore-cluster-lease`